### ISSUE TITLE
query-service - Format "Trace JSON" 

### ISSUE THREAD

**Issue by tiffon** [jaegertracing/jaeger#756]
> Format "Trace JSON" output so the output is nicely readable.

From jaegertracing/jaeger-ui#199

**Comment by yurishkuro** [comment]
I don't think we want to do that by default, but we could have an optional param to the URL that will enable pretty-print
